.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: scala
 
-sudo: required
+jdk:
+- oraclejdk8
 
 scala:
 - 2.11.11
 - 2.12.3
```

Comment:
Maybe we could define an [environment variable](https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml) for the `2.12.3` value.